slapo/schedule.py

```diff
@@ -36,9 +38,18 @@
 )
 from .tracer import trace as trace_module
 from .utils.common import transfer_hooks
+from .utils.mapping import MAPPING_FROM_FUNCTIONAL_TO_MODULE
+from .pattern import Pattern, call
 
 logger = get_logger()
 
+# Wrap call as leaf
+fx.wrap(call)
```

Comment:
It is because of the limitation of torch.fx -- the leaf function wrapper can only be registered in the same module. Otherwise, the wrapper cannot work properly. See https://github.com/pytorch/pytorch/blob/v1.13.1/torch/fx/_symbolic_trace.py#L1011-L1012.

Reply:
I also attach this comment in the file.